src/collective/volto/formsupport/restapi/services/submit_form/post.py

```diff
@@ -303,16 +318,10 @@ def filter_parameters(self):
         """
         do not send attachments fields.
         """
-        skip_fields = [
-            x.get("field_id", "")
-            for x in self.block.get("subblocks", [])
-            if x.get("field_type", "") == "attachment"
-        ]
-        return [
-            x
-            for x in self.form_data.get("data", [])
-            if x.get("field_id", "") not in skip_fields
-        ]
+        for field in self.fields:
+            field.label = field.get_display_value()
```

Comment:
why are you setting data on the same object? just rename get_display_value to label?